src/components/RightFixedNote/RightFixedNoteItem.tsx

```diff
@@ -17,6 +17,8 @@ export type ItemProps = {
   date?: ReactNode
   /** このアイテムの著者 */
   author?: ReactNode
+  /** edit ボタンを表示するかどうか */
+  isEditable?: boolean
```

Comment:
IMO： editable で十分 boolean であることが伝わるので冗長に見えます

```suggestion
  editable?: boolean
```